src/ai/backend/manager/registry.py

```diff
@@ -3653,13 +3654,9 @@ async def get_agent_local_config(
         async with self.agent_cache.rpc_context(agent_id) as rpc:
             return await rpc.call.get_local_config()
 
-    async def purge_images(
-        self,
-        agent_id: AgentId,
-        images: list[str],
-    ) -> PurgeImageResponses:
+    async def purge_images(self, agent_id: AgentId, request: PurgeImagesReq) -> PurgeImageResponses:
```

Comment:
Would `PurgeImagesRequest` be better than `PurgeImagesReq`?
Let’s standardize it one way or the other. @HyeockJinKim

Reply:
Let's standardize all as `Req`, `Resp`.

Reply:
In that case, for `PurgeImageResponseList`, should it be written as `PurgeImageResponses` or `PurgeImageResps`?

Personally, I think using `PurgeImageResponses` for the plural form is easier to understand.

Reply:
`PurgeImagesResponse` (Not multiple responses but multiple images.)

Reply:
What I intended is a list of responses.

```py
@dataclass
class PurgeImageResponses(AbstractAgentResp):
    responses: list[PurgeImageResp]

    @override
    def as_dict(self) -> dict:
        return asdict(self)
```

Reply:
I believe this is a response to an action called `purge images`, not a summary of multiple `purge image` responses.